File: bee/simples/1131.py
def get() -> tuple:
    return map(float, input().split())

def main():
    vitoria1, vitoria2, empate = 0, 0, 0

    while True:
        v1, v2 = get()
        if v1 > v2:
            vitoria1 += 1
        elif v2 > v1:
            vitoria2 += 1
        else:
            empate += 1

        print("Novo grenal (1-sim 2-nao)")
        if int(input()) == 2:
            break

    print(f"{vitoria1 + vitoria2 + empate} grenais")
    print(f"Inter:{vitoria1}")
    print(f"Gremio:{vitoria2}")
    print(f"Empates:{empate}")

    if vitoria1 > vitoria2:
        print("Inter venceu mais")
    elif vitoria2 > vitoria1:
        print("Gremio venceu mais")
    else:
        print("Nao houve vencedor")

main()

File: bee/simples/1131.go
package main

import (
	"fmt"
)

func get() (float64, float64) {
	var v1, v2 float64
	fmt.Scan(&v1, &v2)
	return v1, v2
}

func main() {
	var vitoria1, vitoria2, empate int
	for {
		v1, v2 := get()
		if v1 > v2 {
			vitoria1++
		} else if v2 > v1 {
			vitoria2++
		} else {
			empate++
		}

		fmt.Println("Novo grenal (1-sim 2-nao)")
		var i int
		fmt.Scan(&i)
		if i == 2 {
			break
		}
	}

	fmt.Printf("%d grenais\n", vitoria1+vitoria2+empate)
	fmt.Printf("Inter:%d\n", vitoria1)
	fmt.Printf("Grêmio:%d\n", vitoria2)
	fmt.Printf("Empates:%d\n", empate)

	if vitoria1 > vitoria2 {
		fmt.Println("Inter venceu mais")
	} else if vitoria2 > vitoria1 {
		fmt.Println("Gremio venceu mais")
	} else {
		fmt.Println("Nao houve vencedor")
	}
}
